### PR TITLE
Message error and fix validation to checkbox

### PR DESCRIPTION
**Pull request by williamcosta**
**CHANGELOG** :memo:

* Add `preversion` command to generate package
* Add new message to checkbox field validation
* Add Validation onBlur to checkbox field

**PRINTS** :framed_picture:

🚫

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
     "coverage": "node scripts/test.js --env=jsdom --coverage",
+    "preversion": "git checkout master && git pull origin master && git fetch origin --tags",
```
Comment (iondrimba) [inline package.json]: 💯 🥇

Comment (restarac) [reply package.json]: `preversion` means -> `pullall`?

Comment (williamcosta) [reply package.json]: @restarac the `preversion` command already standard in the community to generate packages in the NPM

File: src/components/Form/validation.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const isValidCellPhone = (value) => {
 
 export const validateField = ({ required, type, value, minLength }) => {
   if (required && value === null) {
-    return errorMessages.REQUIRED_FIELD;
+    return (type === 'checkbox') ? errorMessages.REQUIRED_CHECKBOX_FIELD : errorMessages.REQUIRED_FIELD;
```
Comment (eduardojmatos) [inline src/components/Form/validation.js]: This `(`and `)` are not necessary :)

**Pull request merged**